docs/blog/newsletter-february-2018.rst

```diff
@@ -0,0 +1,128 @@
+
+.. post:: February 5, 2018
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – February 2018
+#########################################
+
+Hi everybody! Kelly O. here – back in the saddle after getting through the whole *OMG, it's a different year now* transition. 
+I'm excited to report that we have a TON of stuff happening in the community. 
+I guess we're starting this new year off strong!
+
```

Comment:
Should "I guess we're starting this new year off strong!" be on a separate line?

Reply:
Nope. But it won't be. 

I'm trying out this weird line break thing that is supposed to make this review process... I dunno, tidier? But imo, it's distracting and annoying, and it breaks lists, so you're not likely to see me do it again. ;)